### DISCUSSION TITLE
Brightness adjustment only works in the first half

### DISCUSSION THREAD

**Discussion by Erik262** [MonitorControl/MonitorControl#745] [Q&A] [answered]
### Before opening the issue, have you...?

- [X] Searched for existing issues
- [X] Looked through [the wiki](https://github.com/MonitorControl/MonitorControl/wiki)
- [X] Updated MonitorControl to the latest version (if applicable)

### Describe the bug

"A picture is worth a thousand words"
![Untitled](https://user-images.githubusercontent.com/11977647/139029096-57600fac-d3a3-45ac-b63d-4b73f6f13c68.gif)

Brightness adjustment only forks in the first half of the "Volumebar"

### Steps to reproduce

I'm using a Monitor (Samsung G9) where ddctl seems to be not working or doesn't have it.

Try to adjust the brightness.

### Expected behavior

Brightness adjusted :)

### Environment Information (please complete the following information)

```markdown
- macOS version: 12.0.1
- Mac model: Hackintosh
- MonitorControl version: 4.0.0
- Monitor(s): Ultrawide Samsung G9
- Apple Silicon/M1 (yes or no): no
```

**Comment by waydabber** [comment, marked as answer]
Hi Erik262,

Thank you for your feedback!

This happens is because "Combine hardware and software dimming" is enabled (under Preferences/General), but the hardware (DDC) component of the control fails with the display. This can either be an issue on the part of the motherboard/GPU hardware (I don't know what card or iGPU use on your Hackintosh), the cabling or the display (maybe DDC is not enabled or the display is incompatible and cannot handle DDC operations).

If the hardware issue cannot be rectified, then I advise you to unclick `Use hardware DDC control` under Displays.

<img width="842" alt="Screen Shot 2021-10-27 at 11 48 21" src="https://user-images.githubusercontent.com/37590873/139042526-6641bd05-feeb-4196-8a06-e461b72759e7.png">

I'll convert this issue into a Discussion as other users might have the same experience.

**Reply by Erik262** [reply]
Great it works! But as far as I know, MonitorControl has never worked for my Monitor. Interestingly it works over HDMI, not over DisplayPort.
